distribution/tools/plugin-cli/src/main/java/org/elasticsearch/plugins/InstallPluginCommand.java

```diff
@@ -532,6 +532,12 @@ private Path stagingDirectoryWithoutPosixPermissions(Path pluginsDir) throws IOE
 
     // checking for existing version of the plugin
     private void verifyPluginName(Path pluginPath, String pluginName, Path candidateDir) throws UserException, IOException {
+        // don't let luser install plugin conflicting with module...
```

Comment:
`luser` -> `user`

Reply:
I'm not going to argue (I pushed 49977e9), but I think it was fine how it (originally) was.